src/features/moviesPage/moviePage/BigMovieTile/index.js

```diff
@@ -0,0 +1,23 @@
+import { imagesApi } from "../../../../core/API";
+import { BackDropWrapper, BackGround, Backdrop, Information, Rate, Rating, Star, Title, Votes } from "./styled";
+import star from "../../../../common/images/star.svg"
+
+const BigMovieTile = ({ poster, title, rate, voteCount }) => (
+    <BackGround>
+        <BackDropWrapper>
+            <Information>
+                <Title>{title}</Title>
```

Comment:
For these titles, you created a separate component in common called "PageHeader" (by the way, here you also need to change the tag to h2 and the name to PageTitle) - why create a new one?